valhalla/baldr/graphreader.h

```diff
@@ -793,6 +794,52 @@ class GraphReader {
    */
   int GetTimezone(const baldr::GraphId& node, const GraphTile*& tile);
 
+  // TODO: temporary until we have real incidents loaded from incident loading singleton
```

Comment:
this is the minimal structure of an incident that i needed to be able to structure the tripleg to have incidents on it